yorkie/src/main/kotlin/dev/yorkie/document/Document.kt

```diff
@@ -105,7 +105,9 @@ public class Document(
     public val presences: StateFlow<Presences> =
         combine(_presences, onlineClients) { presences, onlineClients ->
             presences.filterKeys { it in onlineClients }.asPresences()
-        }.stateIn(scope, SharingStarted.Eagerly, _presences.value.asPresences())
+        }.stateIn(scope, SharingStarted.Eagerly, _presences.value.asPresences()).also {
+            scope.launch { it.collect(::publishPresenceEvent) }
+        }
```

Comment:
```
public val presences: StateFlow<Presences> =
    combine(_presences, onlineClients) { presences, onlineClients ->
        presences.filterKeys { it in onlineClients }.asPresences()
    }.onEach {
        publishPresenceEvent(it)
    }.stateIn(scope, SharingStarted.Eagerly, _presences.value.asPresences())
```

maybe `onEach` could be used instead, timing would be a little different but I don't think it would be a problem.

Reply:
I initially tried to use `onEach` but encountered failing TCs due to timing issues, so I just retained the current implementation.